login_server/src/cpp/JSONInterface/JsonGetLogin.cpp

```diff
@@ -71,11 +71,15 @@ Poco::JSON::Object* JsonGetLogin::handle(Poco::Dynamic::Var params)
 		em->addError(new Error("JsonGetLogin::handle", "generic exception calling userModel->getJson: "));
 		em->sendErrorsAsEmail();
 	}
+	// use both variants for compatibility reasons, but final version is Transactions.pending
+	result->set("Transaction.pending", session->getProcessingTransactionCount());
 	result->set("Transactions.pending", session->getProcessingTransactionCount());
```

Comment:
Why duplicate here with `s` and without?

Reply:
Because I wasn't sure if I have fixed every occurance in community-server, but now I have fixed it